TheMovieDb/Modules/MovieDetail/ViewModel/MovieDetailViewModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MovieDetailViewModel.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 16/11/21.
+//
+
+import UIKit
+
+struct MovieDetailViewModel {
+    
+    let movie: Movie
+    
+    var imageUrl: URL? {
+        let urlImage =  MovieConst.imageCDN + (movie.backdropPath ?? (movie.posterPath ?? ""))
+        return URL(string: urlImage)
+    }
+    
+    var title: String {
+        return movie.title
+    }
+    
+    var overview: String {
+        return movie.overview
+    }
+    
+    var date: String {
+        return "   \(movie.releaseDate ?? "")   "
+    }
+    
+    var popularity: String {
+        return "   \(Int(movie.popularity))%   "
+    }
+    
+    var votes: String {
+        return "   Votes: \(movie.voteCount)   "
+    }
```

Comment:
a small comment on this ones, using this sintaxis for returning strings could be risky `"   \()"` if by any reason you have something like a format emojis or extra characters that are not supported on a string, when trying to use if you could get a error or even a crash if not handled properly, I suggest instead use description property or also string (i.e. this one is available on NSSMutableAttributedString) which returns safely the string removing all non string characters

Reply:
Ok, I will make that change in the last delivery